.circleci/config.yml

```diff
@@ -132,7 +132,9 @@ jobs:
     executor: win32-executor
     steps:
       - checkout
-      - run: apt update && DEBIAN_FRONTEND=noninteractive apt install -y ninja-build python3 python3-pip python3-venv git g++-mingw-w64-i686 zip
+      - run: dpkg --add-architecture i386
+      - run: apt update
+      - run: DEBIAN_FRONTEND=noninteractive apt install -y ninja-build python3 python3-pip python3-venv git g++-mingw-w64-i686 zip wine wine32
```

Comment:
Do we need both `wine` and `wine32`? Or is `wine` a metapackage installing both archs? Asking for the `README.mingw` file...

Reply:
I don’t know to be honest. I copied it from somewhere. It does not look like `wine32` or `wine64` packages depend on `wine` package.